R/FilterStateRange.R

```diff
@@ -456,10 +456,29 @@ RangeFilterState <- R6::R6Class( # nolint
       tagList(
         div(
           class = "choices_state",
+          tags$head(tags$script(
+            HTML(
+              '$(document).ready(function() {
+                      $("[data-toggle=\'popover\']").popover();});'
+            )
+          )),
           div(
-            actionLink(ns("plotly_info"), label = NULL, icon = icon("question-circle")),
+            actionLink(
+              ns("plotly_info"),
+              label = NULL,
+              icon = icon("question-circle"),
+              "data-toggle" = "popover",
+              "data-html" = "true",
+              "data-placement" = "left",
+              "data-trigger" = "hover",
```

Comment:
```suggestion
              "data-trigger" = "click",
```

Reply:
Added additional JS code to close popover if clicked anywhere else on screen